cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/AbstractProgramRuntimeService.java

```diff
@@ -87,15 +87,16 @@
   private final ReadWriteLock runtimeInfosLock;
   private final Table<ProgramType, RunId, RuntimeInfo> runtimeInfos;
   private final ProgramRunnerFactory programRunnerFactory;
-  private final ArtifactRepository artifactRepository;
+  private final ArtifactRepository baseArtifactRepository;
 
-  protected AbstractProgramRuntimeService(CConfiguration cConf, ProgramRunnerFactory programRunnerFactory,
-                                          ArtifactRepository artifactRepository) {
+  protected AbstractProgramRuntimeService(CConfiguration cConf,
+                                          ProgramRunnerFactory programRunnerFactory,
+                                          ArtifactRepository baseArtifactRepository) {
```

Comment:
In all other places I think we are using `noAuthArtifactReposritory`, would be good to change all `base` to `noAuth`